_contrib/new-topic

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -eu
+
+TOPICS_DIR=_topics
+
+if ! [ -d "$TOPICS_DIR" ]
+then
+  echo "Couldn't find $TOPICS_DIR.  Are you sure you're in the repository root directory?"
+  exit 1
+fi
+
+echo -n "Name of new topic (use title case): "
+read
```

Comment:
Perhaps allow this script to take an argument: `new-topic <topic name>`

Reply:
The way I've been running this script is via `make new-topic` which has no easy way to pass the topic name from the `make` invocation to the script (it'd have to be passed as a variable, e.g. `make new-topic TOPIC="Foo Bar Baz"`).  If you still want this (e.g. because you plan to run `_contrib/new-topic` directly), I'm happy to add it.

Reply:
Nah, leave it as it is. It doesn't seem too much extra effort for the script to prompt the user for the topic name.